dnf.spec

```diff
@@ -122,8 +122,10 @@ Requires:       python-libcomps >= %{libcomps_version}
 Requires:       python-librepo >= %{librepo_version}
 %if 0%{?rhel} && 0%{?rhel} <= 7
 Requires:       pygpgme
+Requires:       pygobject3
 %else
 Requires:       python2-pygpgme
+Requires:       python-gobject
```

Comment:
@MichaelMraka 
There probably is some problem with this. (building on 3 but running tests on 2 maybe???)
see: https://copr-be.cloud.fedoraproject.org/results/rpmsoftwaremanagement/rpm-gitoverlay-1479993153.599176/fedora-24-x86_64/00481033-dnf/build.log.gz

```
1:   File "/builddir/build/BUILD/dnf-2.0.0_0.rc1.1-149g47d2fd4/dnf/base.py", line 29, in <module>
1:     from dnf.yum import history
1:   File "/builddir/build/BUILD/dnf-2.0.0_0.rc1.1-149g47d2fd4/dnf/yum/history.py", line 28, in <module>
1:     import gi
1: ImportError: No module named gi
```